### PR TITLE
use logger with minimized cardinality

### PR DESCRIPTION
**Pull request by rreinoldsc**
Moves subset of #print usage to logger

### CODE REVIEW

**Review by anthonyburdi — COMMENTED** [review]
LGTM. Just commenting since it's in draft. Also can we wait to merge until after demos today?

**Comment by anthonyburdi** [comment]
> LGTM. Just commenting since it's in draft. Also can we wait to merge until after demos today?

Also what do you think about making this a pre-release? If you think the change can't wait until next thurs then it definitely can be a release, but we should default to pre-releases for other changes.

**Comment by rreinoldsc** [comment]
@anthonyburdi  I'd like to get it running in our internal agents. If those are on pre-release, then I can make it a pre-release. Otherwise, full release

File: pyproject.toml
Comment (anthonyburdi) [inline pyproject.toml:3]:
```suggestion
version = "20240426.0"
```

**Review by anthonyburdi — APPROVED** [review]
Just need to bump version again but otherwise LGTM.

File: pyproject.toml
Comment (rreinoldsc) [reply pyproject.toml]: i upped it

**Pull request merged**
